test/e2e/util/test_kubernetes.go

```diff
@@ -469,41 +469,12 @@ func (k TestKubernetes) DeleteSecret(secret *v1.Secret) {
 
 // RunOperator runs an operator on a Kubernetes cluster
 func (k TestKubernetes) RunOperator(namespace string) chan struct{} {
-	k.installRBAC(namespace)
```

Comment:
Seems a good cleanup to me. @Crumby do you need the installRBAC func for your tests?

Reply:
@rigazilla let's merge this PR on master
in case of some tests will fails for current release, 2.0.x will be suitable for testing, and for master we will revert this and just fix RBAC creation for scripts only